WalletWasabi.Fluent/Models/Transactions/PrivacyWarning.cs

```diff
@@ -16,6 +16,8 @@ public abstract record PrivacyWarning(WarningSeverity Severity) : PrivacyItem();
 
 public record InterlinksLabelsWarning(LabelsArray Labels) : PrivacyWarning(WarningSeverity.Warning);
 
+public record TransactionKnownAsYoursBy(LabelsArray Labels) : PrivacyWarning(WarningSeverity.Info);
```

Comment:
This is equivalent to `NonPrivateFundsWarning` (just more user friendly) so the severity should be a warning as well.

Reply:
Once https://github.com/zkSNACKs/WalletWasabi/issues/11030#issuecomment-1700547657 is implemented, we can remove the `NonPrivateFundsWarning` when `TransactionKnownAsYoursBy` is active as the second one is less technical.